Code/lib/motor/motor.cpp

```diff
@@ -88,4 +88,20 @@ Drive::~Drive()
 {
     // Turn off motors - When exting the context of this class
     turnOff();
+}
+
+void Drive::accelerate() //Increase pwm value to maximum value
+{
+    for (drive_speed; drive_speed < 255; drive_speed++){
```

Comment:
I'd say to have this function receive a value to increase motor speed from the current. The calling will be done from the main loop. the function should be a utility, the same goes for decelerate function